CVs/input4MIPs_source_id.json

```diff
@@ -121,5 +121,13 @@
         "license_id":"CC BY 4.0",
         "mip_era":"CMIP6Plus",
         "source_version":"1.1.2"
+    },
```

Comment:
Just noting that we need to either delete or deprecate the `UOEXETER-CMIP-1-1-2` which never really existed (same with `UOEXETER-CMIP-0-1-0`)